package.json

```diff
@@ -74,7 +74,7 @@
     "adm-zip": "^0.5.5",
     "archiver": "^5.3.1",
     "archiver-zip-encrypted": "^1.0.11",
-    "bcrypt": "^5.0.0",
+    "bcrypt": "^5.1.0",
```

Comment:
There was a problem with 5.0.0, but can't remember what it was...

(Unrelated to this issue though)